src/layout/section/baseSection/BaseSection.style.tsx

```diff
@@ -0,0 +1,27 @@
+import styled from 'styled-components'
+
+import { componentSizes, horizontalSpace, responsiveBreakpoints } from '../../../_utils/branding'
+
+export const StyledBaseSection = styled.div`
+  & .section-content {
+    padding-left: ${horizontalSpace.global};
+    padding-right: ${horizontalSpace.global};
+  }
+
+  & .section-content--noHorizontalSpacing {
+    padding-left: 0;
+    padding-right: 0;
+  }
+
+  @media (${responsiveBreakpoints.isMediaLarge}) {
```

Comment:
nit breakpoints inside classes to clear reading on what is changing on each part of the component